src/js/index.js

```diff
@@ -35,7 +35,7 @@ store.subscribe(throttle(model.update, 16));
 var ui = require('./components');
 
 // name model Scene
-model.Scene.name = 'Scene';
+model.Scene._name = 'Scene';
```

Comment:
because in this PR he changed .name to ._name with a getter method to retrieve it, unless I am misunderstanding your question.